SPECIFICATIONS.md

```diff
@@ -568,7 +586,9 @@ each block must carry its own version.
 An implementation must refuse a token with a newer format than the range they know.
 An implementation must refuse a token with an older format than the range they know.
 An implementation must always generate tokens at the highest version it can do.
-The current minimum version number is 3.
+
+The current minimum version number is 3.  
+The current maximum version number is 4.
```

Comment:
Here it might be interesting to only use `version = 4` for blocks that have an external sig or explicit scope annotations.

Reply:
I agree, that would make the transition smoother